inorbit_edge_executor/worker_pool.py

```diff
@@ -303,3 +315,38 @@ async def get_mission_status(self, mission_id):
                 return self._workers[mission_id].dump_object()
             else:
                 return None
+
+    async def pause_mission(self, mission_id) -> None:
+        """
+        Pauses a running mission. If there's no worker for the mission it raises
+        MissionNotFoundException(). If the mission is already paused it raises
+        InvalidMissionStateException().
+        """
+        async with self._mutex:
+            if mission_id not in self._workers:
+                serialized_mission = await self._db.fetch_mission(mission_id=mission_id)
+                if serialized_mission and serialized_mission.state["paused"]:
+                    logger.warning(f"Mission {mission_id} is already paused.")
+                    raise InvalidMissionStateException()
+                elif not serialized_mission or serialized_mission.state["finished"]:
+                    logger.warning(f"Mission {mission_id} not found or it's already finished")
+                    raise MissionNotFoundException()
+            else:
+                await self._workers[mission_id].pause()
+                del self._workers[mission_id]
```

Comment:
super nit. I'm not a fan of negating a condition if we are using `else`

```suggestion
            if mission_id in self._workers:
                await self._workers[mission_id].pause()
                del self._workers[mission_id]
            else:
                serialized_mission = await self._db.fetch_mission(mission_id=mission_id)
                if serialized_mission and serialized_mission.state["paused"]:
                    logger.warning(f"Mission {mission_id} is already paused.")
                    raise InvalidMissionStateException()
                elif not serialized_mission or serialized_mission.state["finished"]:
                    logger.warning(f"Mission {mission_id} not found or it's already finished")
                    raise MissionNotFoundException()
```